src/utils/event-utils.ts

```diff
@@ -79,53 +75,9 @@ export const _info = {
     /**
      * This function detects which browser is running this script.
      * The order of the checks are important since many user agents
-     * include key words used in later checks.
+     * include keywords used in later checks.
      */
-    browser: function (user_agent: string, vendor: string | undefined, opera?: any): string {
-        vendor = vendor || '' // vendor is undefined for at least IE9
-        if (opera || _includes(user_agent, ' OPR/')) {
-            if (_includes(user_agent, 'Mini')) {
-                return 'Opera Mini'
-            }
-            return 'Opera'
-        } else if (/(BlackBerry|PlayBook|BB10)/i.test(user_agent)) {
-            return 'BlackBerry'
-        } else if (_includes(user_agent, 'IEMobile') || _includes(user_agent, 'WPDesktop')) {
-            return 'Internet Explorer Mobile'
-        } else if (_includes(user_agent, 'SamsungBrowser/')) {
-            // https://developer.samsung.com/internet/user-agent-string-format
-            return 'Samsung Internet'
-        } else if (_includes(user_agent, 'Edge') || _includes(user_agent, 'Edg/')) {
-            return 'Microsoft Edge'
-        } else if (_includes(user_agent, 'FBIOS')) {
-            return 'Facebook Mobile'
-        } else if (_includes(user_agent, 'Chrome')) {
-            return 'Chrome'
-        } else if (_includes(user_agent, 'CriOS')) {
-            return 'Chrome iOS'
-        } else if (_includes(user_agent, 'UCWEB') || _includes(user_agent, 'UCBrowser')) {
-            return 'UC Browser'
-        } else if (_includes(user_agent, 'FxiOS')) {
-            return 'Firefox iOS'
-        } else if (_includes(vendor, 'Apple') || isSafari(user_agent)) {
-            if (_includes(user_agent, 'Mobile')) {
-                return 'Mobile Safari'
-            }
-            return 'Safari'
-        } else if (_includes(user_agent, 'Android')) {
-            return 'Android Mobile'
-        } else if (_includes(user_agent, 'Konqueror') || _includes(user_agent, 'konqueror')) {
-            return 'Konqueror'
-        } else if (_includes(user_agent, 'Firefox')) {
-            return 'Firefox'
-        } else if (_includes(user_agent, 'MSIE') || _includes(user_agent, 'Trident/')) {
-            return 'Internet Explorer'
-        } else if (_includes(user_agent, 'Gecko')) {
-            return 'Mozilla'
-        } else {
-            return ''
-        }
-    },
+    browser: detectBrowser,
```

Comment:
i moved these functions out of here but left them referenced on the same info object so the caller doesn't have to care